tools/gcs-bucket-mover/gcs_bucket_mover/bucket_mover_service.py

```diff
@@ -33,48 +32,71 @@
 from gcs_bucket_mover import sts_job_status
 
 _CHECKMARK = u'\u2713'.encode('utf8')
+_LOGGER = None
 
 
-def move_bucket(conf):
+def move_bucket(conf):  # Majority statements are logging. pylint: disable=too-many-statements
```

Comment:
too-many-statements is an indication that your function has too much in it and should be split up into smaller functions. It's VERY difficult to test large functions like this, and you really should have unit tests.